index.html

```diff
@@ -1,237 +1,274 @@
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="hidden">See Project</button>
+                </div>
+            </div>
+        </section>
+
+
+        <section id="about-section">
+            <div class="header3">
+                <h2 class="hf">About Me</h2>
+                <p>
+                    Hey there! I can help you build a product, feature or website. Look through some of my work and
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact
+                    me.
+                </p>
+                <button class="button-first">Get My Resume</button>
+            </div>
+            <hr id="hr-abt" />
+            <div class="prog">
+
+                <ul class="list">
+                    <li><img src="./img2/diamond.png" alt=""> <span>
+                            <h2>Language</h2>
+                        </span></li>
+                    </li>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>CSS</li>
+                </ul>
+
+                <ul class="list">
+                    <li><img src="./img2/square.png" alt=""> <span>
+                            <h2>Frameworks</h2>
+                        </span></li>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CabyBara</li>
+                    <li>Selenium</li>
+                </ul>
+                <ul class="list">
+                    <li><img src="./img2/circle.png" alt=""> <span>
+                            <h2>Skills</h2>
+                        </span></li>
+                    <li>CodeKit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li class="l2">Terminal</li>
+                </ul>
+            </div>
+        </section>
+        <!-- section4 -->
+        <section id="contact-area">
+            <div class="forms">
+                <p class="hg">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </p>
+                <form action="https://formspree.io/f/xayzolrz" class="getcontact" method="post">
+                    <input class="input" type="text" name="name" placeholder="Full name" maxlength="30" required />
+                    <input class="input" type="email" name="email" placeholder="Email address" required>
+                    <textarea cols="30" rows="10" class="textarea" name="message" placeholder="Enter text here"
+                        maxlength="500" required></textarea>
+                    <button class="button-final">
+                        Get in touch
+                    </button>
```

Comment:
- [ ] Nice work implementing the highlighted changes from the last review. 👏 . One more thing. The gap between the text and your contact form needs to be adjusted a little it. This presently does not give a great user view and appeal, but with that fixed, you should have your project ready to go.

## Figma:
![image](https://user-images.githubusercontent.com/50990925/222892355-0c67c875-c07b-47a8-af6c-c9ddc595a58c.png)


# Your design:
![image](https://user-images.githubusercontent.com/50990925/222892377-0fdcbab6-4d62-48d5-b5d4-d166782b7f7d.png)